GiniVision/Classes/CameraViewController.swift

```diff
@@ -283,7 +293,11 @@ public typealias CameraErrorBlock = (_ error: CameraError) -> ()
     
     fileprivate func getVideoOrientation(forDeviceOrientation orientation:UIDeviceOrientation) -> AVCaptureVideoOrientation{
         if UIDevice.current.isIpad {
-            return orientationsMapping[UIDevice.current.orientation] ?? .portrait
+            if UIDevice.current.orientation.isFlat { // This only could happen on initialization
```

Comment:
This comment is difficult to understand for me. Either explain it a little better or remove it.

The same thing with the `isFlat` and the mappings repeats. Maybe you should extract it somewhere.